xoa/resource_xenorchestra_backup.go

```diff
@@ -0,0 +1,432 @@
+package xoa
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gofrs/uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vatesfr/xenorchestra-go-sdk/pkg/payloads"
+	v2 "github.com/vatesfr/xenorchestra-go-sdk/v2"
+)
+
+func resourceBackup() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"mode": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"vms": {
+				Type:     schema.TypeList,
+				Required: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+			"schedule": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"cron": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "The cron expression for the backup job schedule.",
+						},
+						"enabled": {
+							Type:        schema.TypeBool,
+							Optional:    true,
+							Default:     false,
+							Description: "Whether the backup job schedule is enabled.",
+						},
+						"timezone": {
+							Type:        schema.TypeString,
+							Optional:    true,
```

Comment:
timezone is required by the jrpc API.

							
```suggestion
							Required:    true,
```